python/kserve/kserve/model_server.py

```diff
@@ -50,6 +53,11 @@
 
 tornado.log.enable_pretty_logging()
 
+class MetricsHandler(RequestHandler):
```

Comment:
took this from https://github.com/globocom/tornado-prometheus/blob/master/tornado_prometheus/__init__.py 

we only needed the metrics handler, when we switch to fastapi, this should be super easy to change.